packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt

```diff
@@ -103,6 +103,9 @@
 # decode_DTS=0x00000000
 # decode_DDP=0x00000000
 
+# enable rc-core driver for RPi-style GPIO IR receiver at GPIO pin 18
+dtoverlay=ir-gpio
```

Comment:
Correct, but I don't have an idea how to solve that. BTW, lirc_rpi needs the same manual intervention.